project/frontend/src/components/SGonksLists/ShortSGonksList/ShortSGonksRow/ShortSGonksRow.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import classes from "./ShortSGonksRow.module.css";
+
+const ShortSGonksRow = (props) => {
+  const getDiffIndicator = () => {
+    const diff =
+      props.sgonk.currentPrice > props.sgonk.priceYesterday
+        ? "+"
+        : props.sgonk.currentPrice < props.sgonk.priceYesterday
+        ? "-"
+        : "=";
```

Comment:
only thing to worry about is if the "prices" are close. Depends on precision you displaying them at. If yesterdays and todays price are same wrt to number of decimal places, it will be strange to show a + or -

Reply:
There will be no decimal places. We're not using floats for currency. Smallest change is 1